front-end/src/components/Helpbtn.tsx

```diff
@@ -1,14 +1,16 @@
 import * as React from "react";
 import Button from "@mui/material/Button";
 import HelpIcon from "@mui/icons-material/Help";
+// plz install mui by :
+// npm install @mui/material @emotion/react @emotion/styled
 
```

Comment:
If we have these packages in the `package.json`, it will get installed on running `npm install` so feel free to remove this comment.

Reply:
thank you for you suggestion